integration-tests/utils/integration-utils.js

```diff
@@ -160,8 +160,9 @@ const select = async (page, labelContains, text) => {
   return click(page, `//div[starts-with(@id, "react-select-") and contains(normalize-space(.),"${text}")]`)
 }
 
-const waitForNoSpinners = page => {
-  return page.waitForXPath('//*[@data-icon="loadingSpinner"]', { hidden: true })
+const waitForNoSpinners = async page => {
+  await page.waitForXPath('//*[@id="root"]')
+  await page.waitForXPath('//*[@data-icon="loadingSpinner"]', { hidden: true })
```

Comment:
`loadingSpinner` is a child element under the `root` element. If `root` element is not loaded in DOM yet, checking `loadingSpinner` would be unexpectedly too soon.